Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -131,21 +136,22 @@ import PayPalCheckout
                             action.set(billingAgreementToken: order.orderID)
                         @unknown default:
                             notifyFailure(with: BTPayPalNativeCheckoutError.invalidRequest, completion: completion)
-
                         }
                     },
                     onApprove: { [weak self] approval in
                         guard let self else {
                             completion(nil, BTPayPalNativeCheckoutError.deallocated)
                             return
                         }
+                        self.clientMetadataID = approval.data.correlationIDs.riskCorrelationID
 
                         tokenize(approval: approval, request: request, completion: completion)
                     },
                     onCancel: {
                         self.notifyCancel(completion: completion)
                     },
                     onError: { error in
+                        self.clientMetadataID = error.correlationIDs.riskCorrelationID
                         self.notifyFailure(with: BTPayPalNativeCheckoutError.checkoutSDKFailed, completion: completion)
```

Comment:
Since we get an `error` in the completion here, should we add it to the `checkoutSDKFailed` error description? It could be helpful for the MXO team to know what error their SDK returned in this callback.

Reply:
Great idea, done!